content/applications/general/payment_acquirers/stripe.rst

```diff
@@ -14,13 +14,31 @@ Configuration
 Credentials tab
 ---------------
 
-Odoo needs your **API Credentials** to connect with your Stripe account, which comprises:
+On premise or on Odoo SH, Odoo needs your **API Credentials** to connect with your Stripe account,
+which comprises:
 
 - :ref:`Publishable Key <stripe/api_keys>`: The key solely used to identify the account with Stripe.
 - :ref:`Secret Key <stripe/api_keys>`: The key to sign the merchant account with Stripe.
 - :ref:`Webhook Signing Secret <stripe/webhook>`: When you enable your webhook on your Stripe
   account, this signing secret must be set to authenticate the messages sent from Stripe to Odoo.
 
+You'll be able to easily create your account by using the **Connect Stripe** button. Once you come
+back on your database from the onboarding :
+
+1) Copy/paste your Publishable and Secret keys (see :ref:`below <stripe/api_keys>`);
+2) Generate your webhook by clicking the **Generate your webhook** button.
+
+On SaaS, click the button **Connect Stripe** and follow the Stripe Onboarding :
+
+1) Fill the Stripe Onboarding information in order to create your account;
+2) Stripe sends an email to validate your email address;
+3) At the end of the process, you're redirected to Odoo. If you completed Stripe onboarding, you're
+   all set and your acquirer is enabled.
```

Comment:
I don't think the enumeration is needed here, also I reversed the first sentence, I think it's better to tell people "what" and then "how"
```suggestion
To create your account, fill in the Stripe Onboarding information. Stripe sends an email to validate
your email address. At the end of the process, you're redirected to Odoo. If you completed the
Stripe onboarding, you're all set and your acquirer is enabled.
```